howto/website-guidelines-checklist.md

```diff
@@ -0,0 +1,41 @@
+## CNCF Website Guidelines
+
+As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md)
+(**Note**, not all of these are applicable to all projects):
+
+- [ ] 1. Website should be [hosted in an open source repo](./repo-setup.md)
+  - [ ] Hosted in the same organization as the main project
+- [ ] 2. Reference the origin company correctly (if needed)
+  *Note*: It is OK to say that, e.g., “Prometheus was originally created by 
+  Soundcloud” or “Kubernetes builds upon 15 years of experience of running 
+  production workloads at Google,” but the origin company should not otherwise
+  be referred to on the project homepage.
+- [ ] 3. No links or forms for capturing enterprise support leads should be present
+  It is fine to have an enterprise support, commercial partners or similar page.
+  - [ ] If page is present, the companies list is alphabetized or randomized on load.
+  - [ ] If page is present, the vetting of companies listed is complete
+    *Note*: Projects are welcome to outsource this vetting to CNCF staff if it
+    becomes a burden.
+- [ ] 4. Links to companies offering support go to a page that at least mentions
+  support of the project
+- [ ] 5. Copyright notice present at bottom of page.
+  Copyright to the project authors or to CNCF, not the origin company.
+- [ ] 6. CNCF Branding elements
+  - [ ] “We are a Cloud Native Computing Foundation project.” or “We are a Cloud
+  Native Computing Foundation sandbox project.” present (depending on status)  
+  - [ ] CNCF logo near the bottom of their project homepage  
+  - [ ] (optional) link to KubeCon + CloudNativeCon as the events approach  
+- [ ] 7. Footer contains:
+  "Copyright The Linux Foundation. All rights reserved. The Linux Foundation has
+  registered trademarks and uses trademarks. For a list of trademarks of The
+  Linux Foundation, please see our
+  [Trademark Usage](https://www.linuxfoundation.org/trademark-usage/) page."
+
+
+## Contributing guidelines and conduct pages
+
+The following files should be in the root of the repository:
+
+- [ ] Code of Conduct
+- [ ] Guidelines for Contributors (CONTRIBUTING.md or similar)
+- [ ] License file(s)
```

Comment:
Governance?

Reply:
I'll add that too.

Reply:
Actually, do we need a GOVERNANCE file for websites, or just the main project repo? I'd think that the governance would be in the main repo, and would be written as a page in the website.

Reply:
Honestly, it's a nice to have! It's the main project repo usually as a markdown file so forget I said anything.

Reply:
I'dd add it in then, thanks!